src/build.rs

```diff
@@ -64,7 +64,12 @@ impl BuildSystem {
         // Ensure the output dist directories are in place.
         fs::create_dir_all(self.cfg.final_dist.as_path())
             .await
-            .with_context(|| "error creating build environment directory: dist")?;
+            .with_context(|| {
+                format!(
+                    "error creating build environment directory: {}",
+                    self.cfg.final_dist.to_string_lossy()
```

Comment:
I would go for `.display()` … but I'll fix this post merge.

Reply:
Can change this. Already did in another place. Btw, there are a few more occasions where `to_string_lossy()` is used when formatting.